integration/docker/Dockerfile

```diff
@@ -26,11 +27,16 @@ RUN cd /opt && \
     (if ls | grep -q ".tar.gz"; then tar -xzf *.tar.gz && rm *.tar.gz; fi) && \
     ln -s alluxio-* alluxio
 
+RUN if [ ${ENABLE_DYANMIC_USER} = "true" ] ; then \
```

Comment:
@apc999 I'm concerned about the security implications. If we really have to give a 777 permission, I think at least there should be warnings in the printout and code comments saying that's high security risk.

Reply:
By default it's disabled, but if the user want to build it for the user which is not in alluxio docker image. They have flexibility to enable it.

Reply:
Yes, and I think in that case at least warnings should be given to the user in some printout (like `echo`?) and in the comment in the Dockerfile.

Reply:
Added